Dockerfile.prod

```diff
@@ -0,0 +1,19 @@
+FROM node:12
+
+ENV HOST 0.0.0.0
+ENV NODE_ENV=development
```

Comment:
We have two NODE_ENV uses on this Dockerfile. Are they necessary?

Reply:
I got a warning saying to not use `NODE_ENV=production` when running `yarn install` for `build`... it's something related to the `eslint`, you can take a look by replacing `developement` by `production`, it will fail on `yarn build` instruction